.kitchen.yml

```diff
@@ -31,7 +31,7 @@ platforms:
       box: windows-8.1
       box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
     provisioner:
-      require_chef_omnibus: 12.0.0
+      require_chef_omnibus: 12
```

Comment:
This should be fixed in windows-guest-support by: https://github.com/test-kitchen/test-kitchen/pull/590

https://github.com/test-kitchen/test-kitchen/commit/b74a96e0add86d6cf1011b91437a55274785dd62